Mathlib/Combinatorics/SimpleGraph/Coloring.lean

```diff
@@ -248,46 +256,72 @@ theorem chromaticNumber_bddBelow : BddBelow { n : ℕ | G.Colorable n } :=
 #align simple_graph.chromatic_number_bdd_below SimpleGraph.chromaticNumber_bddBelow
 
 theorem chromaticNumber_le_of_colorable {n : ℕ} (hc : G.Colorable n) : G.chromaticNumber ≤ n := by
```

Comment:
However this one should stay separate because it can use dot notation
```suggestion
theorem Colorabe.chromaticNumber_le {n : ℕ} (hc : G.Colorable n) : G.chromaticNumber ≤ n := by
```